fetch/api/request/request-reset-attributes.https.html

```diff
@@ -7,6 +7,10 @@
 <script>
 const worker = 'resources/request-reset-attributes-worker.js';
 
+function wait(ms) {
+  return new Promise(resolve => step_timeout(resolve, ms));
```

Comment:
It's not entirely clear to me that using `step_timeout` as a global is appropriate. See #10911.

Reply:
I guess that is not a problem because it won't fail.